etc/kayobe/ansible/gendash.yml

```diff
@@ -0,0 +1,68 @@
+- hosts: localhost
+  vars: 
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    - name: Include the generated kolla.yml
+      include_vars:
+        file: ../../kolla/globals.yml
+        name: services 
+
+    - name: 
+      vars:
+        openstack_services: 
+          - { name: "aodh", backend: "aodh_api_back",  enabled: "{{ services.enable_aodh | default('false') | bool }}" }
```

Comment:
longer term, we could add support for templating an overview dashboard in KA, and we could pull this info from there so we don't have to main this list. Fine for now.

Reply:
`s/services./kolla_/g`